<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="AFSWWI9CYZ2H4A" failures="0" tests="4" name="com.developerscorner.provider.service.AuthServiceTest" time="0.025" errors="0" timestamp="2023-01-26T10:01:18 CST" skipped="0">
  <testcase classname="com.developerscorner.provider.service.AuthServiceTest" name="shouldRegisterAUser" time="0.004"/>
  <system-out/>
  <testcase classname="com.developerscorner.provider.service.AuthServiceTest" name="shouldThrowConflictExceptionIfUserExists" time="0.005"/>
  <system-out/>
  <testcase classname="com.developerscorner.provider.service.AuthServiceTest" name="shouldAuthenticateAUser" time="0.010"/>
  <system-out/>
  <testcase classname="com.developerscorner.provider.service.AuthServiceTest" name="shouldThrowUsernameNotFoundExceptionIfUserNotExists" time="0.006"/>
  <system-out/>
</testsuite> <!-- com.developerscorner.provider.service.AuthServiceTest -->
